smoke-tests/designer/features/pageobjects/pages/formRunner.page.js

```diff
@@ -2,10 +2,45 @@ class FormRunnerPage {
   open(path) {
     return browser.url(`${browser.config.runnerUrl}${path}/`);
   }
+
   get pageTitle() {
     return browser.$("h1");
   }
 
+  /**
+   * Selects a checkbox by label name
+   * @param labelText
+   */
+  selectCheckbox(labelText) {
+    const checkboxIndex = browser
+      .$$(".govuk-checkboxes__label")
+      .findIndex((el) => el.getText() === labelText);
+    return browser.$$(".govuk-checkboxes__input")[checkboxIndex].click();
+  }
+
+  /**
+   * Inputs text into a field that has a specific question
+   * @param labelText
+   * @param fieldValue
+   */
+  inputField(labelText, fieldValue) {
+    let fieldIndex = browser
+      .$$(".govuk-input")
+      .findIndex((el) => el.parentElement().getText().includes(labelText));
+    return browser.$$(".govuk-input")[fieldIndex].setValue(fieldValue);
+  }
+
+  /**
+   * Inputs the address into the provided fields
+   * @param dataObject
+   */
+  ukAddress(dataObject) {
```

Comment:
it looks like the other inputs on here (ie inputDate, inputField, selectRadio, selectFromList) have a verb based suffix, should we have one on here too so it's more obvious that it's going to enter the value?

Reply:
@gstevenson fixed